### DISCUSSION TITLE
Anddroid app only finds peers/ etwork on wifi, and only those on same network

### DISCUSSION THREAD

**Discussion by LibertyFarmer** [markqvist/Reticulum#527] [Help & Questions]
Have sideband wi does running smoothly using wifi and tcp. Finds streams, peers, no problem. On android, only wifi to my own windows peer works. Tried multiple tcp testnet servers, no peers are ever found on android(5g, not wifi).

**Comment by markqvist** [comment]
Are you not seeing any announces, or is communication not working to peers that you know the destination hash (address) of?
